libs/@blockprotocol/graph/src/types/file.ts

```diff
@@ -1,15 +1,18 @@
-import { EntityId } from "./entity.js";
-
-export type FileMediaType = "image" | "video";
+import { Entity } from "./entity.js";
 
 export type UploadFileData = {
-  file?: File | null;
-  url?: string | null;
-  mediaType: FileMediaType;
-};
+  description?: string;
+  name?: string;
+} & ({ file: File } | { url: string });
 
-export type UploadFileReturn = {
-  entityId: EntityId;
-  url: string;
-  mediaType: FileMediaType;
+// @todo -
+export type FileEntityProperties = {
+  "https://blockprotocol.org/@blockprotocol/types/property-type/description/v/1"?: string;
+  "https://blockprotocol.org/@blockprotocol/types/property-type/filename/v/1": string;
+  "https://blockprotocol.org/@blockprotocol/types/property-type/mime-type/v/1": string;
+  "https://blockprotocol.org/@blockprotocol/types/property-type/url/v/1": string;
```

Comment:
Shouldn't these be BaseUri's? (This is why I'd really like `satisfies` on types 🙃 , you want this to be a subtype of `EntityPropertiesObject` I assume, although to be fair `BaseUri` is just a string at the moment so I guess it wouldn't have helped you)